apps/tlon-web/src/components/Sidebar/MessagesSidebar.tsx

```diff
@@ -68,8 +73,17 @@ export default function MessagesSidebar({
         'flex items-center space-x-2 rounded-none',
         messagesFilter === filters.groups && 'bg-gray-50 text-gray-800'
       ),
-    },
+    }
   ];
+  if (cohorts) filterActions.push({
+    key: 'broadcasts',
+    onClick: () => setFilterMode(filters.broadcasts),
+    content: 'Broadcasts',
+    containerClassName: cn(
+      'flex items-center space-x-2 rounded-none',
+      messagesFilter === filters.broadcasts && 'bg-gray-50 text-gray-800'
+    ),
+  });
```

Comment:
This technically works, but it breaks our style conventions. You probably need to run prettier/the linter for all of these changes.